spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/ChannelBindingUtils.java

```diff
@@ -0,0 +1,67 @@
+package org.springframework.cloud.stream.binding;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.beans.factory.BeanFactoryAware;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+import org.springframework.cloud.stream.binder.DirectHandler;
+import org.springframework.integration.config.ConsumerEndpointFactoryBean;
+import org.springframework.integration.scheduling.PollerMetadata;
+import org.springframework.messaging.PollableChannel;
+import org.springframework.messaging.SubscribableChannel;
+import org.springframework.scheduling.support.PeriodicTrigger;
+
+/**
+ * An utility class that provides helper methods for channel binding.
+ * This class needs to be instantiated as a bean so that the configuration properties are available runtime.
+ *
+ * @author Marius Bogoevici
+ * @author David Syer
+ * @author Ilayaperumal Gopinathan
+ */
+public class ChannelBindingUtils implements BeanFactoryAware {
+
+	public static final String SPRING_CLOUD_STREAM_INTERNAL_PREFIX = "spring.cloud.stream.internal";
+
+	public static final String POLLABLE_BRIDGE_INTERVAL_PROPERTY_NAME = SPRING_CLOUD_STREAM_INTERNAL_PREFIX + ".pollableBridge.interval";
+
+	@Value("${" + POLLABLE_BRIDGE_INTERVAL_PROPERTY_NAME + ":1000}")
```

Comment:
I think we should use `DefaultPollerProperties` and by extent `PollerMetadata` set by `ChannelBindingAutoConfiguration` and do away with this.